samples/Azure.ResourceManager.Sample/Generated/VirtualMachineExtensionImageContainer.cs

```diff
@@ -0,0 +1,244 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure;
+using Azure.Core.Pipeline;
+using Azure.ResourceManager;
+using Azure.ResourceManager.Core;
+using Azure.ResourceManager.Resources;
+
+namespace Azure.ResourceManager.Sample
+{
+    /// <summary> A class representing collection of VirtualMachineExtensionImage and their operations over a Subscription. </summary>
+    public partial class VirtualMachineExtensionImageContainer : ArmContainer
+    {
+        private readonly ClientDiagnostics _clientDiagnostics;
+        private readonly VirtualMachineExtensionImagesRestOperations _restClient;
+
+        /// <summary> Initializes a new instance of the <see cref="VirtualMachineExtensionImageContainer"/> class for mocking. </summary>
+        protected VirtualMachineExtensionImageContainer()
+        {
+        }
+
+        /// <summary> Initializes a new instance of VirtualMachineExtensionImageContainer class. </summary>
+        /// <param name="parent"> The resource representing the parent resource. </param>
+        internal VirtualMachineExtensionImageContainer(ArmResource parent) : base(parent)
+        {
+            _clientDiagnostics = new ClientDiagnostics(ClientOptions);
+            _restClient = new VirtualMachineExtensionImagesRestOperations(_clientDiagnostics, Pipeline, ClientOptions, Id.SubscriptionId, BaseUri);
+        }
+
+        /// <summary> Gets the valid resource type for this object. </summary>
+        protected override ResourceType ValidResourceType => Subscription.ResourceType;
+
+        // Container level operations.
+
+        /// <summary> Gets details for this resource from the service. </summary>
+        /// <param name="version"> The String to use. </param>
+        /// <param name="cancellationToken"> A token to allow the caller to cancel the call to the service. The default value is <see cref="CancellationToken.None" />. </param>
+        public virtual Response<VirtualMachineExtensionImage> Get(string version, CancellationToken cancellationToken = default)
+        {
+            using var scope = _clientDiagnostics.CreateScope("VirtualMachineExtensionImageContainer.Get");
+            scope.Start();
+            try
+            {
+                if (version == null)
+                {
+                    throw new ArgumentNullException(nameof(version));
+                }
+
+                var response = _restClient.Get(Id.Parent.Parent.Name, Id.Parent.Name, Id.Name, version, cancellationToken: cancellationToken);
```

Comment:
This will be fixed when we actually have the request path of this resource as a context. Keeping it here, we could have the differences when we actually fix this. Therefore I would like to have this error here, instead of marking this as a non-resource and hiding it.

Reply:
What is the time span before we have the fix? I'm afraid we may have requests of releasing new versions of the SDKs for other features, then we'll need to wait for the fix if this PR gets merged. 

If we can have ETA for totally switching to request path and all agree to not release new SDKs before that, then it is also fine.